ouroboros-network-framework/test/Test/Ouroboros/Network/Driver.hs

```diff
@@ -1,10 +1,12 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE KindSignatures      #-}
 {-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE TupleSections #-}
+{-# LANGUAGE PolyKinds           #-}
```

Comment:
either align all of `#-}` or none.

Reply:
I am actually surprised we need additional extensions, e.g. `PolyKinds`; the removed `runPeerWithByteLimit` didn't seem to require it in this module.

Reply:
The limits with per protocol state types are more complex than the previous byte limit, see `byteLimitsReqResp` bellow.